toil-appliance/Dockerfile.template

```diff
@@ -0,0 +1,22 @@
+FROM mesosphere/mesos-ROLE:1.0.0
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+MAINTAINER CJ Ketchum, cketchum@ucsc.edu
+
+# update apt and install python bits
+RUN apt-get update && apt-get install -y \
+	libffi-dev \
+	python-dev \
+	python-pip
+
+# Mesos interface dependency missing on ubuntu
+RUN pip install protobuf==3.0.0
+
+# Install Toil
+RUN pip install toil[aws,mesos,encryption,cwl]==3.3.0
+
+# Move the Mesos module onto the python path
+RUN cp -r /usr/lib/python2.7/site-packages/mesos /usr/local/lib/python2.7/dist-packages/mesos
```

Comment:
@fnothaft Adding them to the PYTHONPATH didn't work, IIRC because Python's import logic will only look in 1 module for any given import & we unfortunately have 2 modules named mesos we depend on. I can soft link it though, that should work